src/test/java/org/opensearch/neuralsearch/query/HybridQueryBuilderTests.java

```diff
@@ -469,8 +469,7 @@ public void testStreams_whenWrittingToStream_thenSuccessful() {
         NeuralQueryBuilder neuralQueryBuilder = new NeuralQueryBuilder().fieldName(VECTOR_FIELD_NAME)
             .queryText(QUERY_TEXT)
             .modelId(MODEL_ID)
-            .k(K)
-            .vectorSupplier(TEST_VECTOR_SUPPLIER);
```

Comment:
Removed the vectorSupplier since our doWriteTo function cannot write a function to the StreamOutput. Besides this field is calculated during the runtime so we should be good to not have it in this test.